gradle/libs.versions.toml

```diff
@@ -39,7 +39,7 @@ apacheMaven-archiver = "2.5"
 apacheMaven-pluginTools = "3.5.2"
```

Comment:
JFYI I didn't touch anything related to Maven, these versions are very, very old, and I think the chance of breaking something here is considerable. Let's do it when we start improving the Maven runner as a whole